package.json

```diff
@@ -4,11 +4,12 @@
   "description": "",
   "license": "ISC",
   "scripts": {
-    "build": "neutrino build --use neutrino-preset-node",
+    "build": "yarn lint && neutrino build --use neutrino-preset-node && cp src/package.json build && cd build && yarn",
```

Comment:
We probably shouldn't lint here. We should make the build do it and then release using the build.